libraries/chain/include/scorum/chain/evaluators/post_bet_evalulator.hpp

```diff
@@ -3,19 +3,28 @@
 #include <scorum/protocol/scorum_operations.hpp>
 #include <scorum/chain/evaluators/evaluator.hpp>
 
-#include <scorum/chain/dba/db_accessor_fwd.hpp>
-#include <scorum/chain/schema/scorum_objects_fwd.hpp>
+//#include <scorum/chain/dba/db_accessor_fwd.hpp>
+//#include <scorum/chain/schema/scorum_objects_fwd.hpp>
 
 namespace scorum {
 namespace chain {
 
-struct account_service_i;
-struct game_service_i;
+struct data_service_factory_i;
 struct betting_matcher_i;
-struct pending_bet_service_i;
-struct dynamic_global_property_service_i;
 struct betting_service_i;
 
+struct game_object;
```

Comment:
Why not to use scorum_objects_fwd.hpp instead? It allows you not to repeat yourself twice